samples/CommunityToolkit.Maui.Sample/Views/Popups/ToggleSizePopup.xaml.cs

```diff
@@ -0,0 +1,26 @@
+using CommunityToolkit.Maui.Views;
+
+namespace CommunityToolkit.Maui.Sample;
+
+public partial class ToggleSizePopup : Popup
+{
+	readonly Size originalSize;
+
+	public ToggleSizePopup()
+	{
+		InitializeComponent();
+		originalSize = Size;
+	}
+
+	void Button_Clicked(object? sender, System.EventArgs e)
+	{
+		if (originalSize == Size)
```

Comment:
Can be simplified and written in 1 line. But it is not critical. I would prefer readability.